plugins/source/aws/docs/tables/aws_rds_engine_versions.md

```diff
@@ -2,7 +2,7 @@
 
 https://docs.aws.amazon.com/documentdb/latest/developerguide/API_DBEngineVersion.html
 
-The composite primary key for this table is (**account_id**, **region**, **engine**, **engine_version**, **db_engine_version_description**).
+The composite primary key for this table is (**account_id**, **region**, **_engine_version_hash**).
```

Comment:
I think we'll be better off using `(account_id, region, engine, engine_version,  _engine_version_hash)`